<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_supervisor</class>
 <widget class="QMainWindow" name="MainWindow_supervisor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="hotelUnpass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文楷体&quot;;
background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>经营许可证</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店名字</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店特色</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店星级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>拥有者</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>已经上线的酒店: )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>待审批的酒店:（</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>     通过</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>     不通过</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>     下线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      查看详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      退出登录</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="hotelPass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文楷体&quot;;
background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>经营许可证</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店名字</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店特色</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店星级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>拥有者</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="title">
     <string>酒店评价</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>242</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>平均分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="score5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="score4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="score3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="score2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="score1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="scoreaverage">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      顾客管理</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
